cmd/cloudx/project/helper_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2023 Ory Corp
+// SPDX-License-Identifier: Apache-2.0
+
+package project_test
+
+import (
+	"io"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/ory/cli/cmd/cloudx/testhelpers"
+)
+
+type execFunc func(stdin io.Reader, args ...string) (string, string, error)
+
+type bits uint8
+
+const (
+	DefaultProject bits = 1 << iota
+	PositionalProject
+	FlagProject
+)
+
+func runWithProject(t *testing.T, test func(t *testing.T, exec execFunc, projectID string), argPosition bits) {
```

Comment:
I really like this abstraction, but the idiomatic way to handle multiple options is through variadic arguments:

```suggestion
func runWithProject(t *testing.T, test func(t *testing.T, exec execFunc, projectID string), opts ...RunWithProjectOption) {
```

Then you would call it with:
```go
runWithProject(t, func(t *testing.T, exec execFunc, _ string) {
	stdout, _, err := exec(nil, "get", "keto-config", "--format", "json")
	require.NoError(t, err)
	assert.True(t, gjson.Get(stdout, "namespaces").Exists(), stdout)
}, WithDefaultProject, WithPositionalProject)
```

Reply:
cool idea

Reply:
done